modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/apicalannotation/v0.1.0.json

```diff
@@ -0,0 +1,46 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/morphology/apicalannotation/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:ApicalAnnotationShape",
+      "@type": "sh:NodeShape",
+      "label": "Apical annotation shape definition",
+      "targetClass": "nsg:ApicalAnnotation",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0/shapes/ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "prov:used",
+              "description": "The original whole brain cell",
+              "class": "nsg:ReconstructedWholeBrainCell",
```

Comment:
Does this application apply only to nsg:ReconstructedWholeBrainCell ? Can it be applied to any nsg:ReconstructedCell ?